Documentation/api.md

```diff
@@ -277,8 +277,12 @@ PodMetricsEndpoint defines a scrapeable endpoint of a Kubernetes Pod serving Pro
 | params | Optional HTTP URL parameters | map[string][]string | false |
 | interval | Interval at which metrics should be scraped | string | false |
 | scrapeTimeout | Timeout after which the scrape is ended | string | false |
+| tlsConfig | TLS configuration to use when scraping the endpoint. | *[TLSConfig](#tlsconfig) | false |
+| bearerTokenFile | File to read bearer token for scraping targets. | string | false |
```

Comment:
I would not want to introduce this in its raw form like this. This was a mistake we made in the early days of the Prometheus Operator.

Reply:
Changed to a PodMetricsEndpointTLSConfig object